bin/clojupyter

```diff
@@ -0,0 +1,21 @@
+#!/bin/env bash
```

Comment:
Note that this doesn't work on macOS Big Sur (and probably other macOS versions). While I'm no expert on this, I believe a better shebang would be

```suggestion
#!/usr/bin/env bash
```